projects/plugins/jetpack/modules/widgets/gallery.php

```diff
@@ -108,8 +108,7 @@ public function widget( $args, $instance ) {
 				// of logic in that method that shouldn't be duplicated.
 				$carousel = new Jetpack_Carousel();
 
-				// First parameter is $output, which comes from filters, and causes bypass of the asset enqueuing. Passing null is correct.
```

Comment:
Looks like that got changed way back in #5469, when `check_if_shortcode_processed_and_enqueue_assets()` was split out of `enqueue_assets()`.